community-operators/jaeger/jaeger-operator-artifact.yaml

```diff
@@ -0,0 +1,151 @@
+data:
```

Comment:
If you are generating them, then it can be left out.

Reply:
Generating operator-artifacts manually should be fine for now until we have the operator-courier.

Reply:
I am open to having the operator artifacts added to repos if it will make your lives easier to track what you upload to quay. But if we are doing that, then it needs to happen consistently across all operators.

Reply:
I'm open to it as well. We can keep this operator artifact and I can add operator artifacts for all other operators in a separate PR.

Reply:
We can use a separate directory to store all the bundles (one directory in each namespace mapped directory) and make it clear that the operator authors don't need to create it.